### PR TITLE
refactor SymbolFormatter to SymbolPrinter and SymbolFormatter

### PR DESCRIPTION
**Pull request by mfelsche**
and fix printing special operators

### CODE REVIEW

File: sql/src/main/java/io/crate/analyze/relations/RelationPrinter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ */
+
+package io.crate.analyze.relations;
+
+import io.crate.analyze.QueriedTable;
+
+public class RelationPrinter extends AnalyzedRelationVisitor<Void, String> {
+
+    public static final RelationPrinter INSTANCE = new RelationPrinter();
+
```
Comment (mfussenegger) [inline sql/src/main/java/io/crate/analyze/relations/RelationPrinter.java]: can you make the ctor private to force usage of the INSTANCE singleton?

Comment (mfelsche) [inline sql/src/main/java/io/crate/analyze/relations/RelationPrinter.java]: i don't know if my knowledge and experience are that advanced. But maybe there is something on stackoverflow on how to do that. ;)

File: sql/src/main/java/io/crate/analyze/symbol/format/SymbolFormatFunction.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import com.google.common.base.Function;
+import io.crate.analyze.symbol.Symbol;
+
+import javax.annotation.Nullable;
+
+public class SymbolFormatFunction implements Function<Symbol, String> {
+
+    public static final SymbolFormatFunction INSTANCE = new SymbolFormatFunction();
+
```
Comment (mfussenegger) [inline sql/src/main/java/io/crate/analyze/symbol/format/SymbolFormatFunction.java]: add private ctor to force singleton usage.

**Comment by mfussenegger** [comment]
otherwise ack

**Pull request merged**